src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -810,7 +810,7 @@ public static void addToZipFile(final String fileName, final ZipOutputStream zos
             Files.copy(file, zos);
             fis.close();
         }
-        catch (Exception e)
+        catch (final Exception e)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1166)